html/semantics/embedded-content/media-elements/track/track-element/cors/support/cors-tester.py

```diff
@@ -1,49 +1,48 @@
 from wptserve.handlers import HTTPException
-import urllib
 
 def main(request, response):
     if request.method != "GET":
```

Comment:
Does this mean that not all strings need to be annotated?

Reply:
Good catch. We plan to annotate all strings and I missed this one.